packages/core/dist/js-joda.d.ts

```diff
@@ -1549,6 +1550,183 @@ export class YearMonth extends Temporal implements TemporalAdjuster {
+    withYear(year: number): OffsetDateTime
+}
+
+/**
+ * A time with an offset from UTC/Greenwich in the ISO-8601 calendar system, such as `10:15:30+01:00`.
+ *
+ * `OffsetTime` is an immutable date-time object that represents a time, often viewed as
+ * hour-minute-second-offset. This class stores all time fields, to a precision of nanoseconds, as
+ * well as a zone offset. For example, the value "13:45:30.123456789+02:00" can be stored in an
+ * `OffsetTime`.
+ *
+ * This is a value-based class; use of identity-sensitive operations (including reference equality
+ * (`==`), identity hash code, or synchronization) on instances of `OffsetTime` may have
+ * unpredictable results and should be avoided. The `equals` method should be used for comparisons.
+ */
+export class OffsetTime {
+    static FROM: TemporalQuery<OffsetTime>;
+
+    static from(temporal: TemporalAccessor): OffsetTime
+    static now(clockOrZone?: Clock | ZoneId): OffsetTime;
+    static of(time: LocalTime, offset: ZoneOffset): OffsetTime;
+    static of(hour: number, minute: number, second: number, nanoOfSecond: number, offset: ZoneOffset): OffsetTime;
+    static ofInstant(instant: Instant, zone: ZoneId): OffsetTime;
+    static parse(text: string, formatter?: DateTimeFormatter): OffsetTime;
+
+    private constructor();
+
+    adjustInto(temporal: TemporalAdjuster): TemporalAdjuster
+    atDate(date: LocalDate): OffsetDateTime
+    compareTo(other: OffsetTime): number
+    equals(other: any): boolean
+    format(formatter: DateTimeFormatter): string
+    get(field: TemporalField): number
+    hour(): number
+    getLong(field: TemporalField): number
+    minute(): number
+    nano(): number
+    offset(): ZoneOffset
+    second(): number
+    hashCode(): number
+    isAfter(other: OffsetTime): boolean
+    isBefore(other: OffsetTime): boolean
+    isEqual(other: OffsetTime): boolean
+    isSupported(fieldOrUnit: TemporalField | TemporalUnit): boolean
+    minus(amountToSubtract: number, unit: TemporalUnit): OffsetTime
+    minus(amountToSubtract: TemporalAmount): OffsetTime
+    minusHours(hours: number): OffsetTime
+    minusMinutes(minutes: number): OffsetTime
+    minusNanos(nanos: number): OffsetTime
+    minusSeconds(seconds: number): OffsetTime
+    plus(amountToAdd: number, unit: TemporalUnit): OffsetTime
+    plus(amountToAdd: TemporalAmount): OffsetTime
+    plusHours(hours: number): OffsetTime
+    plusMinutes(minutes: number): OffsetTime
+    plusNanos(nanos: number): OffsetTime
+    plusSeconds(seconds: number): OffsetTime
+    query<T>(query: TemporalQuery<T>): T
+    range(field: TemporalField): ValueRange
+    toEpochSecond(date: LocalDate): number
+    toJSON(): string;
+    toLocalTime(): LocalTime
+    toString(): string
+    truncatedTo(unit: TemporalUnit): OffsetTime
+    until(endExclusive: Temporal, unit: TemporalUnit): number
+    with(adjuster: TemporalAdjuster): OffsetTime
+    with(field: TemporalField, newValue: number): OffsetTime
```

Comment:
dito `with` is still missing in `OffsetTime`

Reply:
removed